Source/WebCore/platform/graphics/cg/ImageBufferIOSurfaceBackend.cpp

```diff
@@ -233,7 +233,12 @@ SetNonVolatileResult ImageBufferIOSurfaceBackend::setNonVolatile()
 {
     if (m_volatilityState == VolatilityState::Volatile) {
         setVolatilityState(VolatilityState::NonVolatile);
-        return m_surface->setVolatile(false);
+        auto result = m_surface->setVolatile(false);
```

Comment:
I guess it could! Most backends don't normally support volatility, but I guess having everything support `setVolatileAndPurgeForTesting()` wouldn't be problematic.